src/main/java/csv/ReadCSV.java

```diff
@@ -0,0 +1,31 @@
+package main.java.csv;
+
+import java.io.*;
+import java.util.List;
+
+public class ReadCSV {
+    private final static String PEOPLE_DATA = "people.csv";
+
+    public static void main(String[] args) {
+
+        PersonCSV personCSV = new PersonCSV("", 0, "");
+        List<PersonCSV> personsList = personCSV.getAllPersons();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(PEOPLE_DATA))) {
+            String string;
+            while ((string = bufferedReader.readLine()) != null) {
+                System.out.println(string);
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found");
+            e.printStackTrace();
+        } catch (IOException e) {
+            System.out.println("IOException");
+            e.printStackTrace();
+        }
+
+        String theYoungestPerson = personCSV.getYoungestPerson(personsList);
```

Comment:
Do not use the definite and the indefinite articles in scope of variable naming. (the / a / an)